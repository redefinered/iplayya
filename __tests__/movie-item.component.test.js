/* eslint-disable no-undef */
import React from 'react';
import MovieItem from 'components/movie-item/movie-item.component';
import renderer from 'react-test-renderer';

it('renders if proprty title is "Test Title"', () => {
  const node = renderer.create(<MovieItem />).toJSON();
  expect(node).toMatchSnapshot();
});
