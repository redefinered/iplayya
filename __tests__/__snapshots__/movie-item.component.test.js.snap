// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders if proprty title is "Test Title" 1`] = `
<View
  accessible={true}
  focusable={true}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "marginBottom": 12,
      "marginRight": 10,
      "opacity": 1,
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "rgba(255,255,255,0.1)",
        "borderRadius": 8,
        "height": 170,
        "justifyContent": "center",
        "position": "relative",
        "width": 115,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "rgba(255,255,255,0.1)",
          "borderRadius": 8,
          "height": 170,
          "justifyContent": "center",
          "marginRight": 10,
          "width": 115,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#000000",
              "fontFamily": "System",
              "fontWeight": "400",
              "textAlign": "left",
            },
            Object {
              "color": "white",
              "fontFamily": "iplayyaicons",
              "fontSize": 48,
            },
          ]
        }
      >
        î §
      </Text>
    </View>
  </View>
</View>
`;
